.github/renovate.json5

```diff
@@ -50,13 +50,17 @@
       groupName: "scenes",
       matchPackageNames: ["@grafana/scenes", "@grafana/scenes-react"],
     },
+    {
+      groupName: "faro",
+      matchPackageNames: ["@grafana/faro*"],
+    },
     {
       groupName: "visx",
       matchPackageNames: ["@visx/{/,}**"],
     },
     {
       groupName: "uLibraries",
-      matchPackageNames: ["@leeoniya/ufuzzy", "uplot"],
+      matchPackageNames: ["@leeoniya/**", "uplot"],
```

Comment:
@leeoniya i thought we had a new dependency from you so i tweaked this, but turns out we don't... yet? Are you against you being auto-tagged for these updates?